### ISSUE TITLE
Multimonitor background

### ISSUE THREAD

**Issue by koma111** [tcorreabr/Parachute#15]
Hey, I love this script, it is what I've been looking for for years!
When I saw it, I immediately had to download it.
I have a 1920x1080 primary monitor and a 1280x1024 secondary.
When opening the effect, the windows on the two monitors appear correctly, on their respective screens, but the background on the 1920x1080 is cropped to 1280x1024 and if the two screens have different wallpapers, the wallpaper of the 1280x1024 is shown on both screens in the effect. (See image below).
![Screenshot_20200405_223319](https://user-images.githubusercontent.com/32468934/78509317-7ed1e380-778d-11ea-92df-ea4f414436a3.png)
As you can see, desktop widgets are shown too. I can't decide whether that is good or bad; feature or bug. I leave that to you.

 I know that supporting multiple aspect ratios at the same time is tricky and absolutely not a priority, but I thought I'd let you know, so when you have time later, you can work on this.

Btw OS is Manjaro,
Plasma 5.18.3,
Frameworks 5.68.0,
Qt 5.14.1,
Kernel 5.4.28 64bit,
Intel i3-7100, 8GB RAM.

**Comment by koma111** [comment]
b6788dd seems to have done _something_ to this, but the issue persists:
Using different backgrounds on each screen:
![Screenshot_20200417_213007](https://user-images.githubusercontent.com/32468934/79607405-6ea5f680-80f3-11ea-8da8-ca112ec70c9d.png)
![Screenshot_20200417_213139](https://user-images.githubusercontent.com/32468934/79607425-75cd0480-80f3-11ea-88b6-9308fe0d2daa.png)
Using the same background on both:
![Screenshot_20200417_213218](https://user-images.githubusercontent.com/32468934/79607449-81203000-80f3-11ea-8060-2477c986d4a9.png)
![Screenshot_20200417_213437](https://user-images.githubusercontent.com/32468934/79607456-82e9f380-80f3-11ea-8be4-734ee62b3838.png)

I have just recompiled Parachute, so this is the latest version in Master.
I restarted KWin, and I clicked empty virtual desktops in Parachute on both screens.
Having an 1920x1080 and a 1280x1024 monitor side-by-side is a weird, unique edge-case though, but there are even more complex setups with vertical screens and ultrawides and whatnot. I am just giving this feedback because if you decide to implement full multimonitor support with all the weird configs users come up with, the info might be helpful.

**Comment by tcorreabr** [comment]
It's helpful for sure. But just to check, did you click on all of your desktops outside Parachute?

**Comment by koma111** [comment]
Yes I did. I opened Desktop Grid and clicked every one of them. Still no wallpaper on the 1080p primary screen.

**Comment by koma111** [comment]
Today I did a full restart and it seems to work fine!
![Screenshot_20200418_130557](https://user-images.githubusercontent.com/32468934/79636088-84f49680-8175-11ea-8014-88b59c9c4981.png)
It didn't work yesterday after rebuilding Parachute and logging out and back in. It works today after booting up my computer. My bad if it was fixed already and yesterday I complained for no reason. My bad if I made you work unnecessarily on this. I think it can be closed now, but I can keep playing around with monitors & laptops if you want.
